tests/foreman/ui/test_computeresource.py

```diff
@@ -49,7 +49,7 @@ def rhev_data():
 
 
 @pytest.mark.tier2
-def test_positive_end_to_end(session, rhev_data, module_org, module_location):
+def test_positive_end_to_end(session, rhev_data, module_target_sat):
     """Perform end to end testing for compute resource RHEV.
```

Comment:
@shweta83 the test is using `entities` instead of `module_target_sat.api`. Also, it'd be better if we use `module_target_sat.ui_session()` here. Same suggestion for other tests in the module. It's probably the same in master branch,  so I'll leave it to you to decide if it should be done in separate PR.